src/model/mosaic/MosaicId.ts

```diff
@@ -33,6 +41,18 @@ export class MosaicId {
      */
     public readonly fullName?: string;
 
+    /**
+     * Create a MosaicId for given `nonce` MosaicNonce and `owner` PublicAccount.
+     *
+     * @param   nonce   {MosaicNonce}
+     * @param   owner   {Account}
+     * @return  {MosaicId}
+     */
+    public static createFromNonce(nonce: MosaicNonce, owner: PublicAccount): MosaicId {
+        const mosaicId = MosaicIdentifierGenerator(nonce.nonce, convert.hexToUint8(owner.publicKey));
+        return new MosaicId(mosaicId);
+    }
+
     /**
      * Create MosaicId from mosaic and namespace string id (ex: nem:xem or domain.subdom.subdome:token)
```

Comment:
Update the comment. Search if mosaic ids with the form "namespaceName:mosaicName" appear in other files.

Reply:
Updated in https://github.com/nemtech/nem2-sdk-typescript-javascript/pull/68/commits/d64579052c78767c3db04cb5464860afe657aae0